### PR TITLE
sys-apps/yarn: added rdepend blocker to avoid collision with dev-util/cmdtest

### PR DESCRIPTION
**Pull request by soredake**
Fixes https://bugs.gentoo.org/show_bug.cgi?id=602168

### CODE REVIEW

File: sys-apps/yarn/yarn-0.18.0.ebuild [hunk 1]
```diff
@@ -22,6 +22,7 @@ src_install() {
 	local install_dir="/usr/$(get_libdir)/node_modules/yarn"
 	insinto "${install_dir}"
 	doins -r .
-	dosym "../$(get_libdir)/node_modules/yarn/bin/yarn.js" "/usr/bin/yarn"
+	if use alternative-name; then name="yarnpkg"; else name="yarn"; fi
```
Comment (SoapGentoo) [inline sys-apps/yarn/yarn-0.18.0.ebuild]: make this simpler:
`local name=$(usex alternative-name yarnpkg yarn)`

**Comment by gktrk** [comment]
I am not sure about using a USE flag to change the binary name. Adding a blocker for the colliding package should be sufficient here. e.g. `!!dev-util/cmdtest`

**Comment by mgorny** [comment]
I agree with @gktrk here. Especially that a package named 'yarn' clearly has more claim to the name 'yarn' than `dev-util/cmdtest`.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/77e2d9a...22a4540. Thanks!

**Pull request closed without merging**